Area.h

```diff
@@ -0,0 +1,34 @@
+
+#include "Cineme.h"
+
+
+class Area{
+    private:
+    vector<Cinema> cinema;
+    string nameArea;
+    
+    public:
+    Area(string tmpNameArea){
+        nameArea = tmpNameArea;
+    }
+
+    void addCinema(Cinema name){
+        cinema.push_back(name);
+    }
+
+    void printArea(){
+        cout<<"Area:" << nameArea<<endl; 
+
+    }
+
+    void printAllCinema()
```

Comment:
The name of method shoud be in uppercase.

Reply:
Can be as it is.